internal/cmd/object-storage/bucket/describe/describe.go

```diff
@@ -0,0 +1,113 @@
+package describe
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/object-storage/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/tables"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/objectstorage"
+)
+
+const (
+	bucketNameArg = "BUCKET_NAME"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	BucketName string
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("describe %s", bucketNameArg),
+		Short: "Shows details of an Object Storage bucket",
+		Long:  "Shows details of an Object Storage bucket.",
+		Args:  args.SingleArg(bucketNameArg, nil),
+		Example: examples.Build(
+			examples.NewExample(
+				`Get details of an Object Storage bucket with name "my-bucket"`,
```

Comment:
Here we say `Get details` but in the command description we say `Shows details`. Not sure if this is also the case for all of the other commands, but maybe it would be good to uniformize